.github/workflows/publish.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Twitter, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Publish Twitter Artifacts
+
+on:
+  push:
+    tags:
+      - 'rel/*'
+
+jobs:
+  publish:
+    name: Publish for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            artifact_name: twttr-focus-linux-amd64.tar.bz2
+            asset_name: twttr-focus-linux-amd64.tar.bz2
+          - os: macos-latest
+            artifact_name: twttr-focus-macos-amd64.tar.bz2
+            asset_name: twttr-focus-macos-amd64.tar.bz2
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --release --locked
+    - name: Tar Build
+      shell: bash
+      run: |
+        mkdir release
+        pushd release
+        mkdir bin
+        cp ../target/release/focus bin/focus
+        tar jcf ../${{ matrix.artifact_name }} .
+        popd
```

Comment:
Does this bail if any command fails?

Reply:
it will bail if any command fails, tested on a new branch